src/main/java/seedu/hustler/parser/DateTimeParser.java

```diff
@@ -0,0 +1,128 @@
+package seedu.hustler.parser;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
+
+/**
+ * Handles LocalDateTime conversion to String and vice versa for each add command.
+ */
+public class DateTimeParser {
+
+    /**
+     * Date and time with format d/M/uuuu HHmm.
+     */
+    private static final String DATE_TIME_FORMATTER = "d/M/uuuu HHmm";
+    /**
+     * Date with format d/M/uuuu HHmm.
+     */
+    private static final String DATE_FORMATTER = "d/M/uuuu";
+
+    /**
+     * Convert type String to type LocalDateTime.
+     * @param dateTime date and time of type String.
+     * @return date and time of type LocalDateTime.
+     */
+    public static LocalDateTime getDateTime(String dateTime) {
+
+        LocalDateTime localDateTime;
+
+        try {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_TIME_FORMATTER);
+            localDateTime = LocalDateTime.parse(dateTime,formatter);
+            return localDateTime;
+        } catch (DateTimeParseException e) {
+            System.out.println("Wrong date time format!!!");
+            return null;
+        }
+    }
+
+    /**
+     * Convert type String to type LocalDateTime.
+     * @param date date of type String.
+     * @return date of type LocalDateTime.
+     */
+    public static LocalDateTime getDate(String date) {
+
+        LocalDateTime localDate;
+
+        try {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMATTER);
+            localDate = LocalDateTime.parse(date,formatter);
+            return localDate;
+        } catch (DateTimeParseException e) {
+            System.out.println("Wrong date format!!!");
+            return null;
+        }
+    }
+
+    /**
+     * Convert type LocalDateTime to String.
+     * @param localDateTime date and time of type LocalDateTime.
+     * @return date and time of type String.
+     */
+    public static String convertDateTime(LocalDateTime localDateTime) {
+
+        String dateTime;
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.